changelog/6285.bugfix.rst

```diff
@@ -0,0 +1 @@
+Changed the default scaling for `~sunpy.map.sources.EUIMap` from a linear stretch to a asinh stretch.
```

Comment:
What's the shortest path to restoring the previous behaviour? Especially if we are backporting this we should tell people how to get back to the previous behaviour.

Reply:
```suggestion
Changed the default scaling for `~sunpy.map.sources.EUIMap` from a linear stretch to a asinh stretch.

To revert to the previous linear stretch do the following::

     from astropy.visualization import ImageNormalize, LinearStretch
     euimap.plot_settings["norm"] = ImageNormalize(stretch=LinearStretch())
```